README.md

```diff
@@ -48,7 +48,7 @@ sea-action:
       id: find-node
       run:
         echo "node=$(node -e 'console.log(process.argv[0]);')" >>
-        "$GitHub_OUTPUT"
+        $env:GITHUB_OUTPUT
```

Comment:
Do you know if this works on all the runners or just windows?

The reason I ask is GitHub's documentation has it in the form of `>> "$GITHUB_OUTPUT"`.

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/defining-outputs-for-jobs#example-defining-outputs-for-a-job

The CI job uses the same and it looks like it's working, the CI job does use correct casing though `GitHub` -> `GITHUB`

![image](https://github.com/user-attachments/assets/ec91ae89-8ce9-47d9-a697-54a3e4a0f7a7)

 https://github.com/bryopsida/node-sea-action/blob/main/.github/workflows/ci.yml#L78

Reply:
I tested it on macos/linux/windows on x86 and arm64 :)

Reply:
I can try the proper casing next week to see if that's enough in case this PR seems to risky.

Reply:
Just wanted to be sure it would work for all runners.